scripts/install-deps.sh

```diff
@@ -142,6 +142,32 @@ fetch_and_unpack_range()
         https://github.com/ericniebler/range-v3/archive/refs/tags/0.11.0.tar.gz
 }
 
+fetch_and_unpack_libutempter()
+{
+    fetch_and_unpack \
+        libutempter-1.2.1 \
```

Comment:
the version `1.2.1` is used quite often. for future simplicity, we probably should make that a local variable and have every occurrence replaced by it.